src/sentry_scope.c

```diff
@@ -29,8 +29,10 @@ get_client_sdk(void)
 {
     sentry_value_t client_sdk = sentry_value_new_object();
 
-    sentry_value_t name = sentry_value_new_string(SENTRY_SDK_NAME);
-    sentry_value_set_by_key(client_sdk, "name", name);
+    SENTRY_WITH_OPTIONS (options) {
```

Comment:
Not sure if this is the best way to access the options, open for any feedback! I guess it's safe to assume options are set at this point, right?